public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+
+### A Pure Overflow scenario
+
+It turns out that the architecture of the RPC-4000 does provide for a code layout which would accomplish the feat by using an overflow. Using our simplified bit layout, let's assume that the instruction, at some point, reaches the value:
+
+            0111111CCC
+        MSB <----------> LSB
+
+In this diagram, the opcode doesn't matter, it can be any part of the program logic. The address of the next instruction is `111`, so that's where the next step of the loop is located. The data address is also `111`, which doesn't pose a problem: The instruction may not even need an operand, or the value in the `111` address may be commensurate with the program logic. Normally, the program would proceed to the instruction in location `111`. Now, when we try to increment the data address by `1` (adding `1000`), the "overflow" of the field zeroes out the `(A)` and `(N)` fields, yielding this instruction:
+
+            1000000CCC
+        MSB <----------> LSB
+
+
+Which would execute opcode `CCC` and then jump to address 0, just as Ed Nather wrote. In the above diagram, the index register bit is set to `0`, so that it would toggle to `1` following the address overflow. This toggle may be the origin of Nather's recollection of seeing the bit turned on for no apparent reason.
+
+
+### ...A Less Romantic Alternative
+
+There's another possible scenario, even more compatible with the story and in line with the RPC-4000 specs. This flow is mentioned in [Stassa Patsantzis](https://news.ycombinator.com/item?id=20489273)'s post. It hinges on Ed Nather's partial understanding of the machine's internals. Specifically, two properties of RPC-4000:
+
+- Opcode `23 (10111)` was the machine's _conditional_ `JUMP` instruction, called `TBC` (**T**ransfer on **B**ranch **C**ontrol). This opcode transferred control to the address in the `(A)` field, _If_ an internal switch called the `Branch Control Unit (BCU)` was on. If it was off, the next instruction address would default to the `(N)` field.
+
+![RPC-4000 TBC instruction](https://res.cloudinary.com/dcajl1s6a/image/upload/v1654922031/mels-hack/transfer-branch-control_gc2xg2.png)
+
+- What could switch the `BCU` on? According to the manual - either when a successful comparison had just been made, or - more relevant to our story - following _an overflow_.
+
+![](https://res.cloudinary.com/dcajl1s6a/image/upload/v1655241687/mels-hack/branch-control_xd0vqd.png)
+
+Simply put, conditional branching (e.g. if..else or looping until an index reaches a limit) on the RPC-4000 were implemented with two steps:
+
+1. A test, like comparing two numbers, followed immediately by...
```

Comment:
The whole ellipsis approach doesn't make it easier for the reader. 

Consider pseudo-code, or rephrase. Generally, this should be easier to read.